boa/network.py

```diff
@@ -92,7 +92,7 @@ def _get_capability(self, hex_bytecode):
         try:
             self._rpc.fetch("eth_call", [{"to": None, "data": hex_bytecode}])
             return True
-        except RPCError:
+        except (RPCError, HTTPError):
```

Comment:
`eth_call` can raise an HTTPError now?

Reply:
:thinking: maybe not

Reply:
Yeah it shouldn't, this is not a `debug_` call

Reply:
```suggestion
        except RPCError:
```